matron/src/main.c

```diff
@@ -84,6 +84,7 @@ int main(int argc, char **argv) {
 
     w_init(); // weaver (scripting)
     dev_list_init();
+    dev_virtual_init("Virtual RawMIDI");
```

Comment:
if virmidi module is not loaded, is this going to work?

Reply:
Yes I can confirm it does, it does. Apparently `virmidi` force the kernel to create 4 rawmidi virtual ports of 16 channels each. They exist as long as the system is up. 
While this way (you can read more of it at the end of this documentation https://www.alsa-project.org/alsa-doc/alsa-lib/rawmidi.html) creates a virtual rawmidi port which is open until is close or the program terminated.